analytics/cdk/emr-eks/bin/emr-eks.ts

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env node
+import 'source-map-support/register';
+import * as cdk from 'aws-cdk-lib';
+import EmrEksStack from '../lib/emr-eks-stack';
+import { Stack } from 'aws-cdk-lib';
+import { EmrEksTeamProps } from '../lib/teams/emrEksTeam';
+import { PolicyStatement } from 'aws-cdk-lib/aws-iam';
+
+const app = new cdk.App();
+
+const account = '123456789012';
```

Comment:
Better to get account and region from profile/session instead of hardcoding in the pattern.